djangocms_references/cms_toolbars.py

```diff
@@ -13,16 +15,19 @@ class ReferencesToolbar(CMSToolbar):
     """
 
     def populate(self):
+        opts = References._meta
+        if not self.request.user.has_perm(
```

Comment:
could be `if not ... or obj is None` cause the same thing happens on both ifs.

Reply:
:+1: